src/style.css

```diff
@@ -1 +1,29 @@
-/* add styles here */
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
+  margin: 0;
```

Comment:
Again, be consistent with your margins. If the margin is set to 0 for the `.stars__star` class, ensure it's only applied to the top or bottom unless there's a specific reason to do otherwise.